src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -182,7 +182,8 @@ export class MongoStorageAdapter {
 
   addFieldIfNotExists(className, fieldName, type) {
     return this._schemaCollection()
-      .then(schemaCollection => schemaCollection.addFieldIfNotExists(className, fieldName, type));
+      .then(schemaCollection => schemaCollection.addFieldIfNotExists(className, fieldName, type))
+      .then(() => this.create2dsphereIndex(className, fieldName, type));
```

Comment:
can you rename `create2dsphereIndex` into an umbrella `createIndexesIfNeeded`? more explicit :)